code/game/objects/items/weapons/twohanded.dm

```diff
@@ -381,12 +381,16 @@
 	sharpness = IS_SHARP
 	actions_types = list(/datum/action/item_action/startchainsaw)
 	var/on = 0
+	var/on_text = "it begins to whirr."
+	var/off_text = "the chain stops moving."
+	force_wielded = 21
```

Comment:
if you want to cheat you can just give the chainsaw 100 ap so it will delimb in one hit without messing with on/off